patterns/2-structured/dedicated-community-leader.md

```diff
@@ -56,6 +56,11 @@ A community leader with the properties described above will lend a face and embo
 
 Having excellent and dedicated community leaders is a precondition for the success of InnerSource. It is, however, not a silver bullet. There are many challenges of InnerSource which are above and beyond what a community leader can tackle, such as budgetary, legal, fiscal or other organizational challenges.
 
+## Related Patterns
+
+* [Circle Communities](https://patterns.innersourcecommons.org/p/circle-communities)
+* [Agile InnerSource Dojo](https://patterns.innersourcecommons.org/p/agile-innersource-dojo)
+
```

Comment:
Let's handle the related patterns in a separate PR.

Reply:
Removed.

Reply:
If you agree this is done, please resolve.